.gitignore

```diff
@@ -20,3 +20,4 @@ tmp
 *.o
 *.a
 mkmf.log
+/.idea
```

Comment:
Is this related to the project, or your own dev environment? If it's only for you, I suggest [creating a global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) on your machine.